nodes/generators_extended/receive_from_sorcar.py

```diff
@@ -0,0 +1,86 @@
+import bpy
+import ast
+from bpy.props import StringProperty
+
+from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.data_structure import updateNode
+from sverchok.utils.modules.geom_sorcar import make_sorcar_logo
+
+class SvReceiveFromSorcarNode(bpy.types.Node, SverchCustomTreeNode):
+    ''' Receive Mesh Data From Sorcar '''
+    bl_idname = 'SvReceiveFromSorcarNode'
+    bl_label = 'Receive From Sorcar'
+    sv_icon = 'SV_RECEIVE_FROM_SORCAR'
+
+
+    sc_v, sc_e, sc_f = make_sorcar_logo()
+    sv_v_mask = [[True]*len(sc_v)]
+    sv_e_mask = [[True]*len(sc_e)]
+    sv_f_mask = [[True]*len(sc_f)]
+
+
+    verts: StringProperty(name='Vertices',
+                         default=repr([sc_v]),
+                         options={'ANIMATABLE'})
```

Comment:
animatable is redundant. this is the default state of properties.

Reply:
I copied the code from `generators -> sphere.py` when I started, so I thought this was the standard way Sverchok node properties were made. It also has a one-line description.

Reply:
yep, but it's not so useful to restate the default. i'm not going to retrospectively remove them, but whenever a node gets added or updated i try to nip that in the bud.